docs/source/model_design/vivarium_model_components/GBD_disease_health/index.rst

```diff
@@ -33,11 +33,117 @@ Understanding GBD's Disease and Health Categories
 GBD Definitions
 ---------------
 
-- Include the GBD definitions for each of these different categories 
+These definitions and others can be found on the `glossary Hub page <https://hub.ihme.washington.edu/pages/viewpage.action?spaceKey=INTRANET&title=IHME+Glossary>`_. 
 
-Understanding the GBD Hierarchy
--------------------------------
+.. list-table:: GBD definitions of terms 
+  :widths: 15 15 15
+  :header-rows: 1
 
-- Showing what is a direct subset, what overlaps, what interacts but is independent 
-- Include some information on modeling implications as a result of this 
+  * - Term 
+    - Definition
+    - Notes
+  * - Cause
+    - A cause of death or disability. Includes injuries, diseases, and conditions. 
+    - See cause list for more details 
+  * - Cause List 
+    - The list of all causes in GBD. It is a mutually exclusive and collectively exhaustive list of hierarchical categories that does not ignore any cause of death. All death and disability within GBD can be found within the cause list. 
+    - 
+  * - Health State 
+    - The current state of health associated with a given cause and sequela.
+    - 
+  * - Impairment
+    - The symptoms of a disease, such as vision loss as a result of diabetes. Clinically the same as sequelae. 
+    - GBD calculates these very differently than sequelae despite them being clinically the same. 
+  * - Sequela or Sequelae 
+    - The medical conditions that can occur among people who contract a disease or suffer an injury. In other words, the negative health effects of a cause that are associated with certain health states. For example, chronic kidney disease can be a sequela of diabetes, neck pain can be a sequela of whiplash, and foot ulcers is a sequela of diabetes.
+    - "Sequela" is singular. "Sequelae" is plural. 
 
+
+Understanding the Definitions within the GBD Hierarchy Context
+--------------------------------------------------------------
+
+To understand the above definitions in practice, it is helpful to think of them 
+in the context of a specific example. We will use cardiovascular disease for this. 
+In particular, we will examine: 
+
+- Causes: ischemic heart disease (IHD) and hypertensive heart disease (HHD) 
+- Health states: medically managed, mild, moderate and severe heart failure 
+- Impairments: heart failure 
+- Sequelae: numerous, including "severe heart failure due to IHD" and "AMI first 2 days" among others 
+
+Causes
+++++++
+
+The cause hierarchy diagrams for IHD and HHD are included here as a reference. 
+
+
+**IHD**:
+
+.. image:: cause_hierarchy_ihd.svg
+
+**HHD**: 
+
+.. image:: cause_hierarchy_hhd.svg
```

Comment:
I think it could be worth combining these into a single diagram that includes both causes.

Reply:
I think it could also be worth labeling the "levels" of the hierarchy in the diagram and then noting in the text below that causes are only mutually exclusive/collectively exhaustive at a given level.